x/slashing/errors.go

```diff
@@ -0,0 +1,56 @@
+package slashing
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// Local code type
+type CodeType = sdk.CodeType
+
+const (
+	// Default slashing codespace
+	DefaultCodespace sdk.CodespaceType = 10
+
+	// Invalid validator
+	CodeInvalidValidator CodeType = 201
+	// Validator jailed
+	CodeValidatorJailed CodeType = 202
+)
+
+//nolint
+func ErrNoValidatorForAddress(codespace sdk.CodespaceType) sdk.Error {
+	return newError(codespace, CodeInvalidValidator, "That address is not associated with any known validator")
+}
+
+//nolint
+func ErrBadValidatorAddr(codespace sdk.CodespaceType) sdk.Error {
+	return newError(codespace, CodeInvalidValidator, "Validator does not exist for that address")
+}
+
+//nolint
+func ErrValidatorJailed(codespace sdk.CodespaceType) sdk.Error {
```

Comment:
don't need to have `nolint` infront of all of these - can either put `nolint` above the package definition (as is done in other `errors.go` files) or just don't seperate the error functions with spaces:
```
//nolint
func ErrNoValidatorForAddress(codespace sdk.CodespaceType) sdk.Error {
	return newError(codespace, CodeInvalidValidator, "That address is not associated with any known validator")
}
func ErrBadValidatorAddr(codespace sdk.CodespaceType) sdk.Error {
	return newError(codespace, CodeInvalidValidator, "Validator does not exist for that address")
}
func ErrBlahBalahBlah(codespace sdk.CodespaceType) sdk.Error {
	return newError(codespace, CodeInvalidValidator, "Validator does not exist for that address")
}
```

Reply:
Added to top of file.